src/zampy/datasets/eth_canopy_height.py

```diff
@@ -119,11 +121,13 @@ def ingest(
 
         return True
 
-    def load(
+    def load(  # noqa: PLR0913
         self,
         ingest_dir: Path,
         time_bounds: TimeBounds,
         spatial_bounds: SpatialBounds,
+        resolution: float,
```

Comment:
I guess there won't be a case that the user wants the native resolution of canopy height data, right? Since the resolution of this dataset is quite crazy....by any means, the user always needs to regrid it to be consistent with other datasets, I assume. So we don't even need a default value (which indicates the usage of native resolution) and a fall back function (which does not trigger regridding) for it, right?

Reply:
Yes, exactly. The desired resolution needs to be specified in the recipe. Every dataset will have a different native resolution, so regridding will (basically) always be necessary.